src/Util/Tokens.php

```diff
@@ -606,6 +536,13 @@ final class Tokens
         T_YIELD_FROM   => T_YIELD_FROM,
     ];
 
+    /**
+     * Mapping table for polyfilled constants
+     *
+     * @var array<int, string>
```

Comment:
```suggestion
     * @var array<int, string>
```

No need to change anything, but just pointing out that we can't actually be sure/guarantee that the key will always be an `int` as - just like PHPCS did - other tooling could have polyfilled the tokens with some other type of value.